modules/swagger-codegen-maven-plugin/examples/java-client.xml

```diff
@@ -101,15 +96,16 @@
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
             <version>${junit-version}</version>
-            <scope>test</scope>
+            <!--<scope>test</scope>-->
         </dependency>
     </dependencies>
     <properties>
         <swagger-annotations-version>1.5.0</swagger-annotations-version>
-        <jersey-version>2.12</jersey-version>
-        <jackson-version>2.4.2</jackson-version>
-        <jodatime-version>2.3</jodatime-version>
+        <swagger-core-version>1.5.8</swagger-core-version>
```

Comment:
for swagger core, i believe the latest stable version is 1.5.9: https://mvnrepository.com/artifact/io.swagger/swagger-core